index.js

```diff
@@ -14,15 +16,21 @@ var anymatch = function(criteria, value, returnIndex, startIndex, endIndex) {
   }
   startIndex = startIndex || 0;
   var string = value[0];
+  var altString;
+  if (platform === 'win32' && typeof string === 'string') {
```

Comment:
offhand I can't think of any issues I've had with that logic. @paulmillr can you think of any reasons that `path.sep` wouldn't work here?